audio-worklet/lib/param-controller.js

```diff
@@ -35,7 +35,17 @@ class ParamController {
     this.id_ = options.id || this.name_;
     this.controller_.type = options.type || 'range';
     this.controller_.min = options.min || 0;
-    this.controller_.max = options.max || 0;
+
+    if (options.max == null)
+      this.controller_.max = 1;
+    else
+      this.controller_.max = options.max;
```

Comment:
Good point.  But now the description of options.max (line 28) is wrong because it says its a Number, not Number or null. (Don't know what JSDoc expects, but I certainly interpet it to mean that options.max must be a number, and hence required.)